### PR TITLE
Assign RFC numbers for embedded refract and file source map

### PR DESCRIPTION
**Pull request by kylef**

### CODE REVIEW

**Comment by kylef** [comment]
In future, when these are merged they should be updated.

File: README.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ consistent and controlled path for new features to enter the framework.
 ## Active RFC List
 
 - [0004 Clarity on API Description Namespace](text/0004-clarify-api-namespace.md)
```
Comment (pksunkara) [inline README.md]: @kylef Please move this to completed list

**Comment by pksunkara** [comment]
lgtm. Just a minor issue.

**Comment by kylef** [comment]
Marked RFC0004 as completed now. /c @pksunkara

**Pull request merged**